src/app/pages/rpc-sign-psbt/use-rpc-sign-psbt.tsx

```diff
@@ -107,10 +106,17 @@ export function useRpcSignPsbt() {
         makeRpcSuccessResponse('signPsbt', { id: requestId, result: { hex: bytesToHex(psbt) } })
       );
 
-      // Optional args are handle bc we support two request apis,
-      // but only support broadcasting using the rpc request method
-      if (broadcast && addressNativeSegwitTotal && addressTaprootTotal && fee) {
-        tx.finalize();
+      // Optional args are handled here bc we support two request apis,
+      // but we only support broadcasting using the rpc request method
+      if (broadcast === 'true' && addressNativeSegwitTotal && addressTaprootTotal && fee) {
```

Comment:
It'd be better to turn the string into a boolean in `useRpcSignPsbtParams`. That way we  convert it once, and henceforth use it as a bool, rather than continue to treat it as a string.

Reply:
Ok, thx for feedback, wasn't confident which was best.